setup.py

```diff
@@ -34,4 +57,10 @@ def read(fname):
         'Programming Language :: Python :: Implementation :: CPython',
         'Programming Language :: Python :: Implementation :: PyPy',
     ],
+    cmdclass={
+        'clean': system('rm -rf build dist *.egg-info'),
+        'package': system('python setup.py sdist bdist_wheel'),
+        'publish': system('twine upload dist/*'),
```

Comment:
For publishing to PyPI, I recommend setting up a Travis build stage to run on tagged commits. That way, a release will happen only after CI is passing. See marshmallow for an example: https://github.com/marshmallow-code/marshmallow/blob/b3d9cd3d33f7c9734d62e2224299fecbe3339426/.travis.yml#L28-L43

Not strictly necessary, but it's worked quite nice for a lot of the marshmallow libs.

Reply:
I did this two days ago for flask-rest-api as well, taking inspiration on other marshmallow libs.

https://github.com/Nobatek/flask-rest-api/blob/a0243ea0776e5a68ecf38ac6ba84bff8261795e7/.travis.yml

It took a few trial and error iterations, and my file is probably not optimal but it seems to work.

I'd recommend trying it in a branch first with too additional directives, one to specify the branch and another to aim at test.pypi.org.

Reply:
https://github.com/marshmallow-code/marshmallow-oneofschema/issues/23